KubeArmor/main.go

```diff
@@ -42,12 +42,13 @@ func main() {
 	// options (boolean)
 	enableHostPolicyPtr := flag.Bool("enableHostPolicy", false, "enabling host policies")
```

Comment:
This should be default `true` for KVMA.

Reply:
This can be provided as a param in script for kvma. I suggest not to change the existing value. Others also please suggest